src/lib/parsers/pretty.js

```diff
@@ -51,7 +51,16 @@ export default class PrettyParser {
             indent(2) + _.padEnd(step.keyword + step.name, maxStepLength) + indent(1),
             colorMap[step.result.status]
             );
-          let lineStr = colorize('# ' + featureFile + ':' + step.line, colorMap.comment);
+
+          let stepSource;
+          if (step.match) {
+            stepSource = path.relative(process.cwd(), step.match.location);
```

Comment:
i went with a relative path from the cwd, because the full path can get pretty long